packages/image_picker/image_picker/example/ios/RunnerTests/ImagePickerPluginTests.m

```diff
@@ -146,6 +146,32 @@ - (void)testPluginPickVideoDeviceFront {
                  UIImagePickerControllerCameraDeviceFront);
 }
 
+- (void)testPickMultiImageShouldUseUIImagePickerControllerOnPreiOS14 {
+  if (@available(iOS 14, *)) {
```

Comment:
@jmagman Relevant to our discussions about test matrixes in this repository: we won't ever actually test this in CI currently.